### DISCUSSION TITLE
[Idea] awsome charts collection

### DISCUSSION THREAD

**Discussion by will-ww** [hypertrons/hypertrons-crx#578] [Ideas]
See links below:

- https://github.com/ecmadao/hacknical
- https://github.com/yoshi389111/github-profile-3d-contrib

@zhuxiangning

**Comment by menbotics** [comment]
This issue has not been replied for 24 hours, please pay attention to this issue: @heming6666 @LiuChangFreeman @xiaoya-Esther @tyn1998 @zhuxiangning

**Comment by tyn1998** [comment]
This chart is amazing too:

<img width="1313" alt="image" src="https://user-images.githubusercontent.com/32434520/192417318-fd75f27d-d91c-4c25-b591-3db2a686fcb2.png">

enconter it in https://docs.halo.run/
